src/schema/search.ts

```diff
@@ -47,6 +70,12 @@ export const typeDefs = /* GraphQL */ `
   }
 
   extend type Query {
+    """
+    Get user's search history
+    """
+    searchSessionHistory(limit: Int, lastId: String): SearchSessionConnection!
```

Comment:
Please note that we usually have different signature here. If you check other paginated queries you will see. We accept a cursor as `after` and it comes before the limit if I'm not mistaken